util/LdbcGoStep/pom.xml

```diff
@@ -0,0 +1,74 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.vesoft</groupId>
+  <artifactId>LdbcGoStep</artifactId>
+  <packaging>jar</packaging>
+  <version>2</version>
+  <name>Maven Quick Start Archetype</name>
+  <url>http://maven.apache.org</url>
+  <dependencies>
+      <dependency>
+           <groupId>junit</groupId>
+           <artifactId>junit</artifactId>
+           <version>4.12</version>
+      </dependency>
+      <dependency>
+          <groupId>org.slf4j</groupId>
+          <artifactId>slf4j-api</artifactId>
+          <version>1.7.25</version>
+      </dependency>
+      <dependency>
+          <groupId>com.vesoft</groupId>
+          <artifactId>client</artifactId>
+          <version>2.0.0-beta</version>
```

Comment:
update the version, the interface with the nebula-graph has change

Reply:
I change it to 2.0.0-rc1